lib/signum/engine.rb

```diff
@@ -1,5 +1,12 @@
 module Signum
   class Engine < ::Rails::Engine
     isolate_namespace Signum
+    initializer :append_migrations do |app|
```

Comment:
According to Tom's comment on slack,

```
btw - if we're working on engines, please remove the append_migrations logic from the engine.rb:
initializer :append_migrations do |app|
      unless app.root.to_s.match? root.to_s
        config.paths['db/migrate'].expanded.each do |expanded_path|
          app.config.paths['db/migrate'] << expanded_path
        end
      end
    end
It gives more pain, than the problem it tries to solve 
```
 I guess we should not add @tdegrunt  please confirm and suggest.

Thank you!